lib/KeyboardAwareMixin.js

```diff
@@ -52,21 +52,24 @@ const KeyboardAwareMixin = {
       if (!currentlyFocusedField) {
         return
       }
-
-      UIManager.viewIsDescendantOf(
-        currentlyFocusedField,
-        this.getScrollResponder().getInnerViewNode(),
-        (isAncestor) => {
-          if (isAncestor) {
-            // Check if the TextInput will be hidden by the keyboard
-            UIManager.measureInWindow(currentlyFocusedField, (x, y, width, height) => {
-              if (y + height > frames.endCoordinates.screenY) {
-                this.scrollToFocusedInputWithNodeHandle(currentlyFocusedField)
-              }
-            })
+      if (Platform.OS == 'android') {
+        this.scrollToFocusedInputWithNodeHandle(currentlyFocusedField)
+      }else {
+        UIManager.viewIsDescendantOf(
+          currentlyFocusedField,
+          this.getScrollResponder().getInnerViewNode(),
+          (isAncestor) => {
+            if (isAncestor) {
+              // Check if the TextInput will be hidden by the keyboard
+              UIManager.measureInWindow(currentlyFocusedField, (x, y, width, height) => {
```

Comment:
Only `viewIsDescendantOf` is not available in Android, this could be included in your PR too.